dist/index.js

```diff
@@ -10843,9 +10843,8 @@ module.exports = async function ({ github, context, inputs }) {
   const { opticUrl, npmTag, version, id } = releaseMeta
 
   const run = runSpawn()
+  const branchName = `release/${version}`
   if (!pr.merged) {
-    const branchName = `release/${version}`
-
     logInfo(`deleting ${branchName}`)
```

Comment:
do we need to keep the deletion here too?

Reply:
We've been discussing with @ramonmulia , and most likely we'll delete branches always, independently of whether anything fails.

This approach was prior to the discussion (trying to minimise the need for changes on current tests and workflows). I'm making changes right now.